recipes/pysurfer/meta.yaml

```diff
@@ -0,0 +1,63 @@
+# To update, change version number and sha256 -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+{% set version = "0.7" %}
+{% set sha256 = "c003fa83d7214014e55ce5b831d622acabe935b563810441b991fa1a47ec9fae" %}
+
+package:
+  name: pysurfer
+  version: {{ version }}
+
+source:
+  fn: pysurfer-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pysurfer/pysurfer-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py>=30 or linux]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - imageio
+    - matplotlib
+    - mayavi
+    - nibabel
+    - numpy
+    - scipy
```

Comment:
The reason for these dependencies is that PySurfer checks dependencies in `setup.py`, raising an error if they are not installed (rather than specifying them as deopendencies, because it is assumed that they are easier to install from a source other than PYPI) see https://github.com/nipy/PySurfer/blob/master/setup.py#L38